### PR TITLE
.travis.yml created for Travis-CI Implementation

### PR DESCRIPTION
**Pull request by MichaelTamaki**
Installs scikit, numpy, and librosa without errors.
Test suite returns errors, next step is to fix the tests.

### CODE REVIEW

**Comment by stefanv** [comment]
Closes #3

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+python:
+- "3.5"
+  
+before_install:
+- pip install --no-index --trusted-host travis-wheels.scikit-image.org --find-links=http://travis-wheels.scikit-image.org numpy scipy
+  
+# command to install dependencies
+# install: "pip install -r requirements.txt"
+install: 
+# - pip install numpy
```
Comment (stefanv) [inline .travis.yml]: Remove these two commented lines

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# - pip install scikit
+- pip install librosa
```
Comment (stefanv) [inline .travis.yml]: Add librosa to requirements.txt, and change this line to `pip install -r requirements.txt`.  Then, remove the comment above referring to this instruction.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+# command to run tests
+script: nosetests
```
Comment (stefanv) [inline .travis.yml]: Shall we move to pytest?

**Comment by MichaelTamaki** [comment]
@stefanv Changed the file based off of comments. The current test suite uses nosetests, so it will crash if we run pytest.

**Comment by stefanv** [comment]
You can specify versions in the requirements.txt:

```
librosa==1.6.1
```

Then please open another issue for migrating to the latest librosa (or test, perhaps it already works!)

It's ok to stick with nose for now, we can fix that in another PR.

**Comment by stefanv** [comment]
Does that work, @MichaelTamaki ?

**Comment by MichaelTamaki** [comment]
Found out that librosa 1.6.1 does not exist. The latest version available is 0.4.3. Will need to test which versions work with the code.

**Pull request closed without merging**